### PR TITLE
feat: added Maven java 21 pack

### PR DESCRIPTION
**Pull request by freper138**

### CODE REVIEW

File: packs/maven-java21/charts/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Java 14 application
```
Comment (msvticket) [inline packs/maven-java21/charts/README.md:1]: Update this

**Review by msvticket — CHANGES_REQUESTED** [review]
Almost ready

File: packs/maven-java21/charts/README.md (hunk 1 above)
Comment (freper138) [reply packs/maven-java21/charts/README.md]: Updated to correct java version

**Review by msvticket — APPROVED** [review]

**Pull request merged**